asn1acn.cpp

```diff
@@ -38,9 +38,9 @@
 
 #include <texteditor/texteditorconstants.h>
 
+#include "autocompletion/asnsnippetprovider.h"
```

Comment:
hmmm maybe folder should be named `completion` instead of `autocompletion`. QtCreator distinguish "autocompletion" and "completion assist". We keep both in this folder, so maybe name should be different?